src/components/video actions modal/ActionsModal.jsx

```diff
@@ -0,0 +1,69 @@
+import { useState } from "react";
+import { usePlaylist } from "../../context/playlist-context";
+import classes from "./actionsModal.module.css";
+
+const ActionsModal = ({playlistVideo, setShowModal }) => {
+  const [newPlayList, setNewPlayList] = useState("");
+  const { playlistState, addNewPlaylist,addToPlaylist,removeFromPlaylist } = usePlaylist();
+
+  const addPlaylistHandler = async () => {
+    try {
+      if(newPlayList.trim()===""){
+        return;
+      }
+      let isExist = playlistState.playlists.find(list=> list.title.toLowerCase()=== newPlayList.trim().toLowerCase());
```

Comment:
Since you are not changing `isExist` value, you can use `const` over `let` here, make a practice of using const everywhere, use let only when required.